src/admin.controller.ts

```diff
@@ -98,10 +105,17 @@ export class DefaultAdminController {
     @Req() request: Request,
     @Param() params: AdminModelsQuery,
     @Query('page') pageParam: string = '1',
+    @Query('search') searchParam: string,
   ) {
     const { section, repository, metadata, adminEntity } = await this.getAdminModels(params)
     const page = parseInt(pageParam, 10)
-    const [entities, count] = await repository.findAndCount(getPaginationQueryOptions(page))
+    let query = repository.createQueryBuilder('alias')
+    query = buildPaginationQueryOptions(query, page)
+
+    if (searchParam) {
+      query = buildSearchQueryOptions(query, adminEntity.searchFields, searchParam)
+    }
+    const [entities, count] = await query.getManyAndCount()
```

Comment:
I'd like to not have to worry about the query at all in the controller, this should go in the adminSite